angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -1,120 +1,106 @@
-<div ng-controller="MarketplaceDetailsController as detailsCtrl" class="portlet-frame col-xs-12 col-md-12 portlet-details-page">
-  <div ng-if='loading' class="row details-header no-margin">
-    <loading-gif data-object='obj' data-empty='!loading'></loading-gif>
-  </div>
+<div ng-controller="MarketplaceDetailsController as detailsCtrl" class="portlet-frame portlet-details-page" flex="100">
+    <md-card>
+        <div ng-if='loading' class="row details-header no-margin">
+            <loading-gif data-object='obj' data-empty='!loading'></loading-gif>
+        </div>
+        <div ng-if='error'>
+            <div class="alert alert-danger" role="alert" style='margin: 20px;'>
+                <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+                <span class="sr-only">Error:</span>
+                {{errorMessage}}
+            </div>
+        </div>
 
-  <div ng-if='error'>
-    <div class="alert alert-danger" role="alert" style='margin: 20px;'>
-      <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
-      <span class="sr-only">Error:</span>
-      {{errorMessage}}
-    </div>
-  </div>
+        <!-- DETAILS HEADER -->
+        <md-card-header tabindex="0" layout="row" class="no-margin" ng-if='!loading && !error'>
+            <md-card-avatar layout="column" layout-align="center center">
+                <portlet-icon class="md-headline"></portlet-icon>
+            </md-card-avatar>
+            <md-card-header-text layout="column" layout-align="center start">
+                <span class="md-headline" hide-xs>{{::portlet.title}}</span>
+                <span class="md-title" hide-gt-xs>{{::portlet.title}}</span>
```

Comment:
Perhaps tedious questions, certainly questions about things I am not expert in:

1. Is this a sign that the `md-headline` or `md-title` should itself be "responsive", behave differently at different screen widths? Seems like this is the same text in the same conceptual role that just wants to display differently depending on width.

2. This looks like an either/or construct. Would this be more literate?

```xml
<span class="md-title" show-xs>{{::portlet.title}}</span>
<span class="md-headline" hide-xs>{{::portlet.title}}</span>
```